api/src/utils/user-utils.ts

```diff
@@ -0,0 +1,58 @@
+import { difference } from 'lodash';
+import { isAdmin } from '../middleware/auth';
+import User from '../models/user';
+import { SessionUser } from './helpers';
+
+// All fields in user
+const allFields = Object.keys(User.schema.paths);
+
+const neverViewableFields = ['oauthID', '__v'];
+
+const neverEditableFields = ['_id', 'oauthID', '__v'];
+// Fields that non-Admins and other users cannot view
+const nonViewableFiles = ['phone'];
```

Comment:
I think you meant `nonViewableFields` not files lol

Reply:
Also I was thinking maybe the name `privateFields` is better? I got this confused with `neverViewableFields` at first. And then maybe you can change that to `nonViewableFields`? 

Actually let me just suggest my name suggestions and you can decide to listen to them or not lol. I think these make more sense to me just bc "non" and "never" got a bit confusing and I didn't understand what they were until I read the comments.

`allFields`
`nonViewableFields`
`nonEditableFields`
`privateFields`
`adminEditOnlyFields` or `adminEditableOnlyFields` idk

Reply:
I agree with this sentiment. I was following the exact structure of member db. this is clearer for sure